test/functional/combine_logs.py

```diff
@@ -81,13 +81,14 @@ def read_logs(tmp_dir):
 
     # Find out what the folder is called that holds node 0's debug.log file
     debug_logs = list(pathlib.Path(tmp_dir).glob('node0/**/debug.log'))
-    if len(debug_logs) > 0:
-        assert len(debug_logs) < 2, 'Max one debug.log is supported, ' \
-            'found several:\n\t' + '\n\t'.join([str(f) for f in debug_logs])
-        path = debug_logs[0]
-        chain = re.search(r'node0/(.+?)/debug\.log$', path.as_posix()).group(1)  # extract the chain name
-    else:
-        chain = 'regtest'  # fallback to regtest (should only happen when none exists)
+    match len(debug_logs):
+        case 0:
+            chain = 'regtest' # fallback to regtest
```

Comment:
nit: I don't like it either, but the [rules claim](https://peps.python.org/pep-0008/#inline-comments):
> Inline comments should be separated by at least two spaces from the statement 
```suggestion
            chain = 'regtest'  # fallback to regtest
```

Reply:
Had no idea that was intentionally idiomatic Python. Done.